modules/fx/arrow-fx-kotlinx-coroutines/build.gradle

```diff
@@ -0,0 +1,14 @@
+apply plugin: 'kotlin'
+
+dependencies {
+    implementation project(':arrow-fx')
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$KOTLIN_VERSION"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.3"
```

Comment:
We should put these versions in `gradle.properties` like @rachelcarmena reorganized them